config_webrx.py

```diff
@@ -336,6 +336,36 @@
 # path to the aprs symbols repository (get it here: https://github.com/hessu/aprs-symbols)
 aprs_symbols_path = "/usr/share/aprs-symbols/png"
 
+# Uncomment the following to customize gateway beacon details reported to the aprs network
+#   Plese see Dire Wolf's documentation on PBEACON configuration for complete details:
+#   https://github.com/wb2osz/direwolf/raw/master/doc/User-Guide.pdf 
+
+# Symbol in its two-character form as specified by the APRS spec at http://www.aprs.org/symbols/symbols-new.txt
+#   Default: Receive only IGate (do not send msgs back to RF)  
+# aprs_igate_symbol = "R&"
+
+# Custom comment about igate 
+#   Default: OpenWebRX APRS gateway
+# aprs_igate_comment = "OpenWebRX APRS gateway"
+
+# Antenna Power, Height, Gain 
+#   Unspecified by default
+# Transmitter power (0 by default as only RX is supported)
+# aprs_igate_power = "0"
+# Antenna height in feet
+# aprs_igate_height = "20"
+# Antenna gain in dBi
+# aprs_igate_gain = "0"
+# Antenna direction (N, NE, E, SE, S, SW, W, NW).  Omnidirectional by default
+# aprs_igate_dir = "NE"
+
+# Frequency info to contact you by voice
+#   Unspecified by default
+# aprs_igate_freq = "146.955"
+# aprs_igate_tone = "74.4"
+# aprs_igate_offset = "-0.60"
```

Comment:
this is a gateway, and there's no contacting it. if you want a personal beacon, you should send one separately. these parameters should not be present in a gateway beacon.

Reply:
Agree, same analysis as 'aprs_igate_power'.

Reply:
Removed from config file, but left in logic for now as an unlisted option.

Reply:
Same. Please remove!

Reply:
done.